metadata.json

```diff
@@ -34,8 +34,8 @@
     }
   ],
   "dependencies": [
-    { "name": "puppetlabs/stdlib", "version_requirement": ">= 4.1.0 < 5.0.0" },
-    { "name": "puppetlabs/nodejs", "version_requirement": ">= 0.6.1 < 2.0.0" },
-    { "name": "proletaryo/supervisor", "version_requirement": ">= 0.4.0" }
+    { "name": "puppetlabs/stdlib", "version_requirement": ">= 4.1.0 < 5.0" },
```

Comment:
At least with Puppet 3.7.4 on OS X, this causes upon `puppet module build puppet-ghost`:

```
Error: Invalid 'version_range' field in metadata.json: Unparsable version range: ">= 4.1.0 < 5.0"
```

I believe that the lack of support for two field version numbers is probably a bug in Puppet itself, but it [appears](https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#dependencies-in-metadatajson) to only support three field versions, hence e02f7f4e50151d15818471f70c70e7095a4604d2.